src/styles/blocks/index.scss

```diff
@@ -0,0 +1,36 @@
+html {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  margin: 0 auto;
+
+  min-width: 230px;
+  max-width: 550px;
+
+  font: 14px "Helvetica Neue", Helvetica, Arial, sans-serif;
+  font-weight: 300;
+  background: #f5f5f5;
+  color: #4d4d4d;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+
+button {
```

Comment:
It's recommended to create a specific class for buttons to avoid applying these styles to all buttons globally.

Reply:
Done.